### DISCUSSION TITLE
Changing Interface Colors

### DISCUSSION THREAD

**Discussion by edwardp3** [lxqt/lxqt#2266] [Q&A] [answered]
On one of the ML's I'm on, a question was asked about changing the color of the interface and I'm curious about this myself. 

For example, in an e-mail app. when selecting an entry in the message list, a blue bar appears (in most cases). I have several virtual machines installed and with the exception of two where this same bar is orange, the others are all blue. Is there an LXQt color setting that would change that color, or is that something distro or app-specific that can't be changed?

I thought it might be Appearance / Widget Style / (under Qt Palette) Selection, which looks to be the same color, but once this setting was changed in one of the VM's, then the app re-launched, the color afterwards was still the same shade of blue.

Thanks in advance.

**Comment by tsujan** [comment, marked as answer]
> For example, in an e-mail app. when selecting an entry in the message list, a blue bar appears...

Apps may be written in Qt or GTK (Qt Palette is only for Qt), and if they're written in Qt, they can have hard-coded colors (= colors in their codes).

**Reply by edwardp3** [reply]
These are GTK apps, not Qt. I figured they may have been hard-coded. 

Thanks.

**Reply by tsujan** [reply]
> These are GTK apps,

Blue is the selection color of Adwaita, the default GTK theme.

**Reply by edwardp3** [reply]
I'm using my Debian Testing VM right now and both the GTK2 and 3 Themes were on Numix by default, which provides an orange appearance. Changing that to a different theme, provided a different color. At least I was on the correct screen. 

Now to see what other themes might be available from the distros.  :)
